src/palace/manager/feed/acquisition.py

```diff
@@ -660,18 +661,16 @@ def single_entry(
                 _work, active_license_pool, active_edition, identifier, annotator
             )
         except UnfulfillableWork as e:
-            logging.info(
+            cls.logger().info(
                 "Work %r is not fulfillable, refusing to create an <entry>.",
                 work,
+                exc_info=e,
             )
             return cls.error_message(
                 identifier,
                 403,
                 "I know about this work but can offer no way of fulfilling it.",
             )
-        except Exception as e:
-            logging.error("Exception generating OPDS entry for %r", work, exc_info=e)
-            return None
```

Comment:
Drop this overly broad `except`. If we are getting random exceptions, we should handle them as they come up. Looking back at our logs, I don't see this being triggered recently, so it should be fine to drop.